CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## [0.0.4] - November 22, 2022
+
+* [PR#15](https://github.com/SimformSolutionsPvtLtd/flutter_chatview/pull/15) 
```

Comment:
Remove PR from start and add description to this if it is a bug fix. If its a new feature then no need to provide PR link and just add description.

Reply:
Resolved